.github/workflows/docs.yml

```diff
@@ -0,0 +1,33 @@
+name: mobile_docs
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - v*
+  pull_request:
+
+jobs:
+  docs:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 10
```

Comment:
we might want to give it a longer timeout as it seems like contention can slow things down